src/System.IO.FileSystem.AccessControl/src/System/Security/AccessControl/FileSecurity.cs

```diff
@@ -180,7 +180,7 @@ public FileSystemRights FileSystemRights
         internal static int AccessMaskFromRights(FileSystemRights fileSystemRights, AccessControlType controlType)
         {
             if (fileSystemRights < (FileSystemRights)0 || fileSystemRights > FileSystemRights.FullControl)
-                throw new ArgumentOutOfRangeException(nameof(fileSystemRights), SR.Format(SR.Argument_InvalidEnumValue, fileSystemRights, "FileSystemRights"));
+                throw new ArgumentOutOfRangeException(nameof(fileSystemRights), SR.Format(SR.Argument_InvalidEnumValue, fileSystemRights, nameof(AccessControl.FileSystemRights)));
```

Comment:
`FileSecurity` has the `FileSystemRights` property, which will be `nameof`ed instead of the `FileSystemRights` class.